Modules/Networking/Requests/DataUploadRequest.swift

```diff
@@ -59,7 +60,8 @@ public extension DataUploadRequest {
       province: String,
       exposureDetectionSummaries: [CodableExposureDetectionSummary],
       maximumExposureInfoCount: Int,
-      maximumExposureDetectionSummaryCount: Int
+      maximumExposureDetectionSummaryCount: Int,
+      countriesOfInterest: [Country] = []
```

Comment:
Did you check that this doesn't break the traffic analysis mitigation strategies?